src/Features/Traits/Cookies.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\BLT\Features\Traits;
+
+use Behat\Gherkin\Node\TableNode;
+use Illuminate\Contracts\Cookie\QueueingFactory as CookieFactory;
+use PHPUnit\Framework\Assert;
+
+trait Cookies
+{
+    use Application;
+    use HttpRequest;
+
+    /**
+     * @When I set cookie :name with value :value
+     * @When I create a cookie named :name with value :value
+     * @When I add a cookie :name with value :value
```

Comment:
Maybe `@When cookie :name with value :value is set` would more consistent with other tests?

Reply:
Done

Reply:
deleteCookie() has the same thing.